Source/WebKit/Shared/Cocoa/CoreIPCNSURLProtectionSpace.serialization.in

```diff
@@ -20,7 +20,7 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-#if PLATFORM(COCOA)
+#if PLATFORM(COCOA) && !HAVE(WK_SECURE_CODING_NSURLPROTECTIONSPACE)
```

Comment:
Can we write like this:
> #if PLATFORM(COCOA)
> #if HAVE(WK_SECURE_CODING_NSURLPROTECTIONSPACE)
> ...
> #else
> ...
> #endif // HAVE(WK_SECURE_CODING_NSURLPROTECTIONSPACE)
> #endif // PLATFORM(COCOA)

Reply:
The generator doesn't support the `else` style yet.